src/vl-map-search.js

```diff
@@ -123,19 +125,45 @@ export class VlMapSearch extends vlElement(HTMLElement) {
       this.__searchEventListenerRegistered = true;
       this._selectElement.addEventListener('search', (event) => {
         if (event && event.detail && event.detail.value) {
-          fetch(this.searchUrl + event.detail.value).then((response) => {
-            return response.json();
-          }).then((data) => {
-            if (data && data.SuggestionResult) {
-              const resultaten = data.SuggestionResult.map((resultaat) => {
-                return {
-                  value: resultaat,
-                  label: resultaat,
-                };
-              });
-              this._selectElement.choices = resultaten;
-            }
-          });
+          const lambertCoordinaat = LambertCoordinaat.of(event.detail.value);
+
+          if (lambertCoordinaat === undefined) {
+            fetch(this.searchUrl + event.detail.value).then((response) => {
```

Comment:
Ik zou string concatenatie proberen vermijden bij het opstellen van URLs (bv. `new URL(...)` gebruiken) en ook zien dat je waar nodig url encoding toepast.

Reply:
vind dit zelf ook niet zo super, maar dit stond er al zo. Voor de locatie url heb ik dit afgeschermd in een aparte methode. Je zou ook met string replace kunnen werken voor params. Heb ik ook al gezien in het verleden.